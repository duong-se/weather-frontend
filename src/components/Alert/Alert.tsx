import React from 'react'

export type AlertProps = {
  message: string
}

export const Alert: React.FC<AlertProps> = ({ message }) => {
  return <div className="alert alert-danger w-100">{message}</div>
}

export default Alert
