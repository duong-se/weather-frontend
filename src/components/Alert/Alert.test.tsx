import { shallow } from 'enzyme'
import React from 'react'
import Alert from './Alert'

describe('Alert', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Alert message="mockMessage" />)
    expect(wrapper).toMatchSnapshot()
  })
  it('should render message', () => {
    const mockMessage = 'mockMessage'
    const wrapper = shallow(<Alert message={mockMessage} />)
    const messageText = wrapper.find('.alert').text()
    expect(messageText).toEqual(mockMessage)
  })
})
