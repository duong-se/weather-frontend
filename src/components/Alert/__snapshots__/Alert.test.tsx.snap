// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should match snapshot 1`] = `
<div
  className="alert alert-danger w-100"
>
  mockMessage
</div>
`;
